src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
```

Comment:
Ideally, this should be as simple as possible.  I don't think people are going to know what to put here.  I like the idea of making it automatic - otherwise, the default value will always be empty string.

My vote would be to automatically set the test name just like we do for microbenchmarks.  I like the grouping mechanism that @dpodder suggests above.